### PR TITLE
settings-migration scripts

### PR DESCRIPTION
**Pull request by jessegoodier**
new migration scripts

### CODE REVIEW

File: settings-migration/upload-settings.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+fi
+if [ -f $OUTPUT_DIR/asset-reports.json ]; then
+  kubectl cp -n $NAMESPACE -c aggregator $OUTPUT_DIR/asset-reports.json ${kubecost_aggregator}:/var/configs/asset-reports.json
+  echo "Uploaded asset-reports.json"
+fi
+if [ -f $OUTPUT_DIR/budgets.json ]; then
+  kubectl cp -n $NAMESPACE -c aggregator $OUTPUT_DIR/budgets.json ${kubecost_aggregator}:/var/configs/budgets.json
+  echo "Uploaded budgets.json"
+fi
+if [ -f $OUTPUT_DIR/cloud-configurations.json ]; then
```
Comment (srpomeroy) [inline settings-migration/upload-settings.sh:30]: Does this live on the aggregator pod? I only see it on the cost-analyzer pod in kc-demo.

**Comment by jessegoodier** [comment]
@avrodrigues5 did I miss any paths? 

I tested this against 2.3.x to 2.4.0

**Review by cliffcolvin — APPROVED** [review]

**Comment by chipzoller** [comment]
Removal of `copy-reports.sh` is now causing the link checker to break in kubecost/docs as this is referenced in the documentation at `using-kubecost/navigating-the-kubecost-ui/reports.md`.

**Comment by jessegoodier** [comment]
> Removal of `copy-reports.sh` is now causing the link checker to break in kubecost/docs as this is referenced in the documentation at `using-kubecost/navigating-the-kubecost-ui/reports.md`.

https://github.com/kubecost/docs/pull/1136

**Pull request merged**